tests/dummy/app/models/cake.js

```diff
@@ -0,0 +1,7 @@
+import DS from 'ember-data';
+
+export default DS.Model.extend({
```

Comment:
Will remove this and the logic in `index.js` once I'm done manually testing.

Reply:
sounds good 👍